beacon_node/lighthouse_network/src/service/utils.rs

```diff
@@ -62,20 +60,17 @@ pub fn build_transport(
     // yamux config
     let mut yamux_config = libp2p::yamux::YamuxConfig::default();
     yamux_config.set_window_update_mode(libp2p::yamux::WindowUpdateMode::on_read());
+    let multiplexer = core::upgrade::SelectUpgrade::new(yamux_config, mplex_config);
+    let (transport, bandwidth) = transport
+        .upgrade(core::upgrade::Version::V1)
```

Comment:
In general (not just due to this upgrade): Consider using `Version::V1Lazy` for 0-RTT protocol negotiation.

Reply:
thanks, this one is new to me, I'll be reading about this